src/main/java/uk/gov/hmcts/ccd/domain/service/createdraft/DefaultSaveDraftOperation.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.ccd.domain.service.createdraft;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.ApplicationParams;
+import uk.gov.hmcts.ccd.data.draft.DefaultDraftRepository;
+import uk.gov.hmcts.ccd.data.draft.DraftRepository;
+import uk.gov.hmcts.ccd.domain.model.std.CaseDataContent;
+import uk.gov.hmcts.ccd.domain.model.draft.CaseDataContentDraft;
+import uk.gov.hmcts.ccd.domain.model.draft.CreateCaseDataContentDraft;
+import uk.gov.hmcts.ccd.domain.model.draft.Draft;
+
+import javax.inject.Inject;
+
+@Service
+@Qualifier("default")
+public class DefaultSaveDraftOperation implements SaveDraftOperation {
+
+    private final DraftRepository draftRepository;
+    private final ApplicationParams applicationParams;
+
+    protected static final String CASE_DATA_CONTENT = "CaseDataContent";
+
+    @Inject
+    public DefaultSaveDraftOperation(@Qualifier(DefaultDraftRepository.QUALIFIER) final DraftRepository draftRepository,
+                                     ApplicationParams applicationParams) {
+        this.draftRepository = draftRepository;
+        this.applicationParams = applicationParams;
+    }
+
+    @Override
+    public Draft saveDraft(final String uid,
+                           final String jurisdictionId,
+                           final String caseTypeId,
+                           final String eventTriggerId,
+                           final CaseDataContent caseDataContent) {
+        CaseDataContentDraft caseDataContentDraft = buildCaseDataContentDraft(uid, jurisdictionId, caseTypeId, eventTriggerId, caseDataContent);
+        CreateCaseDataContentDraft createCaseDataContentDraft = new CreateCaseDataContentDraft(caseDataContentDraft, CASE_DATA_CONTENT, applicationParams.getDraftMaxStaleDays());
+        return draftRepository.set(createCaseDataContentDraft);
```

Comment:
The 3 lines above are a bit complicated to read. I believe this is due to the number of parameters passed in 2 function calls and the name of the variables assigned being quite similar.

Maybe a builder pattern could be used to create an instance of `CreateCaseDataContentDraft` without exposing the underlying `CreateCaseDataContentDraft`?

`CaseDataContentDraft` and `CreateCaseDataContentDraft` could be renamed to be more distinguished. At the moment, having `buildCaseDataContentDraft` followed by `new CreateCaseDataContentDraft` almost feel like the same thing is done twice.